### ISSUE TITLE
Add link to existing item in Detected Devices view

### ISSUE THREAD

**Issue by SourceCodeSourcerer** [NetHome/NetHomeServer#18]
It would be useful to have a link to the indicated (and found) home item. It is now only shown as "Existing".

**Comment by stefangs** [comment]
Good point. It is a bit tricky, because sometimes there are many. Currently ONH does not keep track on which items consumed an event, I have tried to keep the event processing as light weight as possible
